RInAppMessaging/Classes/CampaignDispatcher.swift

```diff
@@ -75,14 +76,24 @@ internal class CampaignDispatcher: CampaignDispatcherType {
                     Error: Campaign (\(campaign.id)) does not exist in the repository anymore (race condition?). Proceeding with old data...
                     """)
             }
-            self.router.displayCampaign(campaign) { [weak self] in
+
+            let campaignTitle = campaign.data.messagePayload.title
+            self.router.displayCampaign(campaign, confirmation: {
+                let contexts = campaign.contexts
+                guard let delegate = self.delegate, !contexts.isEmpty, !campaign.data.isTest else {
```

Comment:
What is `isTest`property?

Reply:
to mark test campaigns. It's true when you use "Send to test device" button in the dashboard